### ISSUE TITLE
Implement all the Amazon API operation

### ISSUE THREAD

**Issue by masterT** [t3chnoboy/amazon-product-api#75]
We should implement all the Amazon API operations. We could do a milestone?

**Comment by t3chnoboy** [comment]
I hope you meant amazon product advertising api, because amazon api is huge 😅

**Comment by masterT** [comment]
Yes off course 😋, anyway there is already an official module for the AWS.

**Comment by JuanCrg90** [comment]
i like the idea, i have a lot of time without check the codebase, but i'd like to help.

already exists official package @masterT?

**Comment by masterT** [comment]
There is an [official module for the Amazon Web Service (AWS)](https://github.com/aws/aws-sdk-js), but there isn't for the Amazon Product Advertising API.
